isis/src/cassini/apps/cisscal/cisscal.xml

```diff
@@ -229,6 +229,10 @@
       updated values in divideByAreaPixel, and added sensitivity vs
       time correction. Fixes #3351.
     </change>
+    <change name="Adam Paquette" date="2019-11-06">
+      Moved output cube instantiation from the beginning of the script to after
+      the dark subtraction and bit weight corrections have been performed.
```

Comment:
Add the github issue number.

Reply:
Was this moved after these corrections or is it after the calculation/setup for these steps?

Reply:
It is after those steps are performed